fissa/tests/test_core.py

```diff
@@ -244,32 +244,147 @@ def test_prepfirst(self):
 
     def test_redo(self):
         exp = core.Experiment(self.images_dir, self.roi_zip_path, self.output_dir)
+        capture_pre = self.capsys.readouterr()  # Clear stdout
         exp.separate()
+        capture_post = self.recapsys(capture_pre)
+        self.assert_starts_with("Doing", capture_post.out)
+        capture_pre = self.capsys.readouterr()  # Clear stdout
         exp.separate(redo_prep=True, redo_sep=True)
+        capture_post = self.recapsys(capture_pre)
+        self.assert_starts_with("Doing", capture_post.out)
+
+    def test_load_cache(self):
+        image_path = self.images_dir
+        roi_path = self.roi_zip_path
+        exp1 = core.Experiment(image_path, roi_path, self.output_dir)
+        exp1.separate()
+        exp = core.Experiment(image_path, roi_path, self.output_dir)
+        # Cache should be loaded without calling separate
         actual = exp.result
         self.assert_equal(len(actual), 1)
         self.assert_equal(len(actual[0]), 1)
         self.assert_allclose(actual[0][0], self.expected_00)
 
-    def test_noredo(self):
+    def test_load_cache_piecemeal(self):
```

Comment:
These tests could each do with a short docstring explaining their purpose, as they're all a bit archaic

Reply:
Okay, I added docstrings for the new/changed tests.